ngraph_bridge/ngraph_pipelined_tensors.cc

```diff
@@ -132,8 +125,10 @@ void PipelinedTensorsStore::return_tensors(size_t id) {
 PipelinedTensorVector PipelinedTensorsStore::get_group(bool is_input,
                                                        size_t i) {
   PipelinedTensorVector group;
-  for (size_t idx = 0; idx < (is_input ? m_num_inputs : m_num_outputs); idx++) {
-    group.push_back((is_input ? m_in_tensors : m_out_tensors)[idx][i]);
+  if (is_input) {
+    group = m_in_tensors[i];
+  } else {
+    group = m_out_tensors[i];
```

Comment:
Lets get rid of `PipelinedTensorVector group` and return in the if/else itself to get rid of an extra copy